settings.py

```diff
@@ -125,6 +125,7 @@
     'django.template.loaders.eggs.Loader',
 )
 
+# Add this to localsettings and set it to False, so that CSRF protection is enabled on localhost
 CSRF_ALWAYS_OFF = True
```

Comment:
what do you think of renaming this to something like `CSRF_PROTECTION_ENABLED` or `PREVENT_CSRF`? the current name implies that CSRF is *on sometimes* (whatever that means) if the value is true